modules/monitoring-configuring-remote-write-storage.adoc

```diff
@@ -21,15 +21,25 @@ Doing so has no impact on how or for how long Prometheus stores metrics.
 * You have installed the OpenShift CLI (`oc`).
 * You have set up a remote write compatible endpoint (such as Thanos) and know the endpoint URL.
 See the link:https://prometheus.io/docs/operating/integrations/#remote-endpoints-and-storage[Prometheus remote endpoints and storage documentation] for information about endpoints that are compatible with the remote write feature.
-* You have set up authentication credentials for the remote write endpoint. 
+* You have set up authentication credentials in a `Secret` object for the remote write endpoint.
+The secret needs to be created in the same namespace as the Prometheus object for which you configure remote write: `openshift-monitoring` for default platform monitoring or `openshift-user-workload-monitoring` for user workload monitoring.
+ 
 +
 [CAUTION]
 ====
-To reduce security risks, avoid sending metrics to an endpoint via unencrypted HTTP or without using authentication.
+To reduce security risks, avoid sending metrics to an endpoint using unencrypted HTTP or without using authentication.
 ====
 
 .Procedure
 
+Follow these steps to configure remote write for default platform monitoring in the `cluster-monitoring-config` config map in the `openshift-monitoring` namespace.
+
+[NOTE]
+====
+If you want to configure remote write for the Prometheus instance that monitors user-defined projects, make similar edits to the `user-workload-monitoring-config` config map in the `openshift-user-workload-monitoring` namespace. 
+If you do so, note that the Prometheus config map component is called `prometheusK8s` in the `cluster-monitoring-config` `ConfigMap` object and `prometheus` in the `user-workload-monitoring-config` `ConfigMap` object.
```

Comment:
```suggestion
If you do so, note that the Prometheus config map component is called `prometheusK8s` in the `cluster-monitoring-config` config map and `prometheus` in the `user-workload-monitoring-config` config map.
```

Reply:
Revised to read `Note that the Prometheus config map component is called `prometheus` in the `user-workload-monitoring-config` `ConfigMap` object and not `prometheusK8s`, as it is in the `cluster-monitoring-config` `ConfigMap` object.`